### PR TITLE
[Fuzzing] oss-fuzz support for fuzzing

### PR DESCRIPTION
**Pull request by 0x34d**
[OSS-Fuzz](https://google.github.io/oss-fuzz) fuzzing  for continuous fuzzing of [wt](https://github.com/emweb/wt), to find memory related security bugs.

### CODE REVIEW

**Comment by MatthiasVC** [comment]
Hi,

my apologies for the late reply.
After some internal discussion, we will first try this out locally, before we fully integrate this. You'll likely hear from me soon (expect to be notified next week).

Best,
Matthias

**Comment by 0x34d** [comment]
> Hi,
> 
> my apologies for the late reply. After some internal discussion, we will first try this out locally, before we fully integrate this. You'll likely hear from me soon (expect to be notified next week).
> 
> Best, Matthias

@MatthiasVC ping

**Comment by MatthiasVC** [comment]
Please see my above review remark.

**Comment by MatthiasVC** [comment]
My apologies, I simply forgot to submit the comment(s).

File: fuzz/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,13 @@
+INCLUDE_DIRECTORIES(${WT_SOURCE_DIR}/src)
+
+ADD_EXECUTABLE(fuzz-css  fuzz-css.C)
+ADD_EXECUTABLE(fuzz-eval fuzz-eval.C)
+ADD_EXECUTABLE(fuzz-json fuzz-json.C)
+ADD_EXECUTABLE(fuzz-uri  fuzz-uri.C)
+ADD_EXECUTABLE(fuzz-xml  fuzz-xml.C)
+
+TARGET_LINK_LIBRARIES(fuzz-css  PRIVATE wt $ENV{LIB_FUZZING_ENGINE})
```
[the remark this answers is not in the document]
Comment (0x34d) [reply fuzz/CMakeLists.txt:9]: but why are you running `introspector` ?

**Comment by 0x34d** [comment]
> My apologies, I simply forgot to submit the comment(s).

Shit happens.

File: fuzz/CMakeLists.txt (hunk 1 above)
[the remark this answers is not in the document]
Comment (MatthiasVC) [reply fuzz/CMakeLists.txt:9]: I was looking at this page: https://google.github.io/oss-fuzz/further-reading/clusterfuzz/,
I was under the impression that this was how part of the output for the statistics on the fuzzers was generated, am I wrong?

[the remark this answers is not in the document]
Comment (0x34d) [reply fuzz/CMakeLists.txt:9]: >I was looking at this page: https://google.github.io/oss-fuzz/further-reading/clusterfuzz/,
>I was under the impression that this was how part of the output for the statistics on the fuzzers was generated, am I wrong?

This is internal documentation; you don't have to worry about it.
Our part includes `ASAN`, `MSAN`, `UBSAN`, and `Coverage`.

`MSAN` and `UBSAN` are not working in this project due to a bug in the Boost library.

**Comment by MatthiasVC** [comment]
Right, this seems good for me to merge then.
I'll only ask to change our primary email from `info@emweb.be` to `wt-security@emweb.be`.

Also, are you set on also receiving the emails? We would prefer to keep any issues internal first. Of course if you are happy to receive them, I won't stop you.

**Comment by MatthiasVC** [comment]
Ok, I then think it's acceptable to merge this!

Thank you for the patience.

**Pull request merged**